### PR TITLE
chromium: update to 81.0.4044.122

### LINKED ISSUE #369: chromium: update to 81.0.4044.122
Stable release announcements since our latest update:

- https://chromereleases.googleblog.com/2020/04/stable-channel-update-for-desktop_21.html

Targets and platforms we need to check:

**chromium-x11:**
- [ ] zeus, clang, MACHINE="intel-core2-32"
- [ ] zeus, clang, MACHINE="intel-corei7-64"
- [ ] warrior, clang, MACHINE="raspberrypi3"
- [ ] warrior, clang, MACHINE="intel-corei7-64"

**chromium-ozone-wayland:**

- [ ] zeus, clang, MACHINE="m3ulcb"
- [ ] zeus, clang, MACHINE="qemuarm64"
- [ ] warrior, clang, MACHINE="raspberrypi3"

PS We will no longer test for thud, but for Zeus instead.

### PR DESCRIPTION
**Pull request by msisov**
Stable release announcements since our latest update:
https://chromereleases.googleblog.com/2020/04/stable-channel-update-for-desktop_21.html

Test-built with:

* chromium-x11
  * zeus, MACHINE="intel-corei7-64"
  * zeus, MACHINE="intel-core2-32"
  * warrior, MACHINE="intel-corei7-64"
* chromium-ozone-wayland:
  * zeus, MACHINE="qemuarm64"
  * zeus, MACHINE="m3ulcb"
  * warrior, MACHINE="raspberrypi3"

License changes

Removed licenses:
* build/fuchsia/fidlgen_js/third_party/enum34/LICENSE:
  - https://crrev.com/c/2007393 - Removed fidlgen_js code, tests, build & bot rules.
* third_party/libjpeg/LICENSE
  - https://crrev.com/c/2078742 - Removed jpeg decoder
* third_party/sfntly/COPYING.txt
  - https://crrev.com/c/1716256 - Removed sfntly

Added/Changed licenses:
* third_party/harfbuzz-ng/src/COPYING
  - https://github.com/harfbuzz/harfbuzz/commit/5440313924172e155e34391f033f5e6c5e2390b3#diff-7116ef0705885343c9e1b2171a06be0e Updated COPYING.

Added licenses:
* net/third_party/quiche/src/LICENSE
  - https://quiche.googlesource.com/quiche.git/+/b0827d9978c36efd3ad8fe14b60abae1078ea6d1 Added README and other files.
* third_party/android_ndk/NOTICE
  - https://chromium-review.googlesource.com/c/android_ndk/+/1754382 - Upgrade to NDK r20
* third_party/libgifcodec/LICENSE.md
  - https://skia-review.googlesource.com/c/libgifcodec/+/254420  - Added LICENSE file.

Fixes #369
Signed-off-by: Maksim Sisov <msisov@igalia.com>

### CODE REVIEW

**Comment by msisov** [comment]
gentle reminder

File: recipes-browser/chromium/chromium-gn.inc [hunk 1]
```diff
@@ -11,8 +11,6 @@ B = "${S}/${OUTPUT_DIR}"
 SRC_URI += " \
         file://0001-Do-not-use-nullptr-initalization-of-fwd-declared-typ.patch \
         file://0002-media-do-not-use-fwd-decl-with-nullptr-instantiation.patch \
-        file://0001-BookmarkModelMerger-Move-RemoteTreeNode-declaration-.patch \
-        file://0001-cros-search-service-Include-cmath-for-std-pow.patch \
```
Comment (rakuco) [inline recipes-browser/chromium/chromium-gn.inc:15]: It looks like you forgot to `git rm` these two files.

**Comment by msisov** [comment]
@rakuco what's the purpose of adding the links to the patches that added/removed/changed those license files? Does anybody use that?

**Comment by rakuco** [comment]
> @rakuco what's the purpose of adding the links to the patches that added/removed/changed those license files? Does anybody use that? I used to be doing that, but decided to skip that this time as I fail to see the purpose of such thing.

I don't know if anyone does that, to be honest. I used to do because it didn't take too much time and it felt like it could be useful to others who might need to do that kind of work downstream.

**Comment by msisov** [comment]
Ok, waiting for @otavio to provide feedback about providing links for licence changes and if that is actually required.

**Comment by rakuco** [comment]
See also: https://github.com/OSSystems/meta-browser/pull/103#issuecomment-371928654

File: recipes-browser/chromium/chromium-gn.inc (hunk 1 above)
Comment (msisov) [reply recipes-browser/chromium/chromium-gn.inc:15]: done

**Comment by msisov** [comment]
> See also: [#103 (comment)](https://github.com/OSSystems/meta-browser/pull/103#issuecomment-371928654)

ok. done

**Comment by otavio** [comment]
> Ok, waiting for @otavio to provide feedback about providing links for licence changes and if that is actually required.

We don't need the change link but we need a summary. Specially because as chromium has a huge number of included libraries it is hard to keep track of those.

**Pull request merged**